lsstCam/crosstalk/README.md

```diff
@@ -0,0 +1,4 @@
+Crosstalk coefficients for LSSTCam, `c0` and `c1`. The latter is the non-linear term. 
+The coefficents were provided by Adam Snyder from laboratory spots measurements. 
+The script used to make the `ecvs` files here can be found in:
+`$OBS_LSST_DATA_DIR/python/lsst/obs/lsst/script/write_non_linear_crosstalk_coeffs.py`
```

Comment:
This should also note that the sources are available in the USDF butler.  If anything else was needed to run the script other than setting up the appropriate packages, that should be mentioned here as well.
And, a typo I didn't notice until just now: that should be `$OBS_LSST_DIR`.